internal/util/cli/table/simple/simple.go

```diff
@@ -117,7 +117,7 @@ func profileStatusLayout(table *tablewriter.Table) {
 func ruleEvaluationsLayout(table *tablewriter.Table) {
 	defaultLayout(table)
 	table.SetHeader([]string{
-		"Rule ID", "Rule Name", "Entity", "Status", "Remediation", "Entity Info"})
+		"Rule Type ID", "Rule Name", "Entity", "Status", "Remediation", "Entity Info"})
```

Comment:
Spotted this during local testing. "Rule ID" is confusing - the table is in fact showing a rule type ID here.